packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -1020,9 +1019,7 @@ function commitUnmount(
       if (enableDeprecatedFlareAPI) {
         unmountDeprecatedResponderListeners(current);
       }
-      if (enableDeprecatedFlareAPI || enableUseEventAPI) {
-        beforeRemoveInstance(current.stateNode);
-      }
+      beforeRemoveInstance(current.stateNode);
```

Comment:
Don't we need this for `enableDeprecatedFlareAPI` still?

Reply:
We don't need the flag anymore. The code that Flare used (beforeblur stuff) was moved out, but I forgot to remove this flag.